sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventHubsEventSource.cs

```diff
@@ -191,7 +191,7 @@ private EventHubsEventSource()
 
         /// <summary>
         ///   Indicates that a client is closing, which may correspond to an <see cref="EventHubConnection" />,
-        ///   <see cref="EventHubProducerClient" />, <see cref="EventHubConsumerClient" />, or <see cref="EventProcessorClient" />.
+        ///   <see cref="EventHubProducerClient" />, <see cref="EventHubConsumerClient" />, or <c>EventProcessorClient</c>.
```

Comment:
Why exactly are we making this change? Is that because we'll be moving it to another package?

Reply:
Yup.  You can't reference `cref` instances across packages.    So, we're still calling it out and referring to the new package via `<seealso`